db/seeds.rb

```diff
@@ -6,12 +6,26 @@
   password: 'kakaka22'
 )
 
-group = ::Group::Group.create(
+group = ::Group::Group.create!(
   name: 'Tomatika'
 )
 
-::Group::Membership.create(
+::Group::Membership.create!(
   user: ::Group::User.find(user.id),
   group: group,
   role: ::Group::Membership::ROLES[:admin]
 )
+
+producer = ::Producers::Producer.create!(
+  name: 'Joan Pipa',
+  email: 'joan.pipa@katuma.org',
+  address: 'pl. Catalunya 1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.